docs/book/install-and-build.sh

```diff
@@ -54,9 +54,9 @@ esac
 
 # grab mdbook
 # we hardcode linux/amd64 since rust uses a different naming scheme and it's a pain to tran
-echo "downloading mdBook-v0.3.1-${arch}-${target}.${ext}"
+echo "downloading mdBook-v0.4.3-${arch}-${target}.${ext}"
```

Comment:
Want to pull the version into a var and reuse it?

Reply:
Done!